kedro/extras/datasets/pandas/gbq_dataset.py

```diff
@@ -137,12 +137,18 @@ def _describe(self) -> Dict[str, Any]:
 
     def _load(self) -> pd.DataFrame:
         sql = "select * from {}.{}".format(self._dataset, self._table_name)  # nosec
-        return pd.read_gbq(
-            sql,
-            project_id=self._project_id,
-            credentials=self._credentials,
-            **self._load_args
-        )
+        if "query" not in self._load_args:
+            self._load_args.update({"query": sql})
```

Comment:
I think you can shorten this/make it cleaner/more readable by just doing `self._load_args.setdefault("query", sql)` :)

https://docs.python.org/3/library/stdtypes.html#dict.setdefault